### ISSUE TITLE
How do you use MixPanel with different flavors and build types?

### ISSUE THREAD

**Issue by colabug** [mixpanel/sample-android-mixpanel-integration#9]

**Comment by patedit** [comment]
Could you please elaborate?

**Comment by marcoRS** [comment]
There is different App stores on Android such as the Play Store vs Amazon App Store. The App is essentially the same but there could be data that only makes sense for one app vs the other if one app store supports features. 

Can one use the same key/setup but separate out data when needed by package name?

**Comment by patedit** [comment]
You could definitely have a project token as a build param which will be different for different flavors and resolve that token at runtime.

**Comment by marcoRS** [comment]
Yeah for sure, I'm aware of that setup my question is whether its best practice to separate by token or if one can collect the data from all apps with the same tokens and just filter out within the Mixpanel dashboard.

**Comment by patedit** [comment]
Ah, got it. That's a great question. At Mixpanel we used to encourage having different projects projects for different apps. We now have a product in beta called Workspaces where you can create filter views within a project. In your case, you could create two workspaces: one per flavor and do the analysis using the same project token.
